2022/en/src/K06-broken-authentication.md

```diff
@@ -30,6 +31,8 @@ Service account (SA) tokens can be presented to the Kubernetes API as an authent
 
 ***Authenticate users and external services using short-lived tokens:*** All authentication tokens should be as short-lived as tolerable. This way if (and when) a credential is leaked, it is possible that it may not be replayed in the time necessary to compromise the account. 
 
+![Broken Authentication - Mitigations](/assets/images/K06-2022-mitigation.gif)
```

Comment:
It looks like some of the recommendations in the illustration are the opposite of the markdown text...

Reply:
There is a missing 'not' in "Service account tokens should **not** be used from outside the cluster". Great catch!

Reply:
Recommendations are not inline with the markdown text. Marking as resolved.